src/html/index.html

```diff
@@ -178,6 +183,48 @@ <h2 class="card__heading card__heading--error"><img src="{{{ cdn }}}/images/circ
             <hr>
             <button type="button" class="button button--text-only card__back button--unpadded" data-handler="go-to-initial-page">Need to log in with a different account?</button>
           </div>
+          <div id="error-general" data-screen hidden>
```

Comment:
@hidde what's the current deal with the error msgs? Are these just sitting here unused in this version?

Reply:
Woops, they were. Have just removed them.